src/Frontend/Themes/Bootstrap/src/Js/lib/share-btns.js

```diff
@@ -0,0 +1,123 @@
+;(function ( $, window, document, undefined ) {
+
+	"use strict";
+
+		var pluginName = "shareBtns";
+
+		/**
+		 * Add Share Buttons on any DOM element.
+		 * Simple set the data-share attribute to the social network you
+		 * want to share to.
+		 * @example
+		 * <a href="#" data-share="facebook">Share on Facebook</a>
+     * Use data-url to share an alternative url, otherwise the current page is shared
+     * <a href="#" data-share="facebook" url="http://www.sumocoders.be">Share on Facebook</a>
+     * Twitter has extra options:
+     * <a href="#" data-share="twitter" via="sumocoders" hashtags="sumocoders" text="Check this out!">Share on Twitter</a>
+		 */
```

Comment:
is it just me or is your indentation really strange? (throughout the whole file)